src/Gui/DlgPreferencesImp.cpp

```diff
@@ -782,11 +782,9 @@ void DlgPreferencesImp::expandToMinimumDialogWidth()
     auto screen = windowHandle()->screen();
     auto availableSize = screen->availableSize();
 
-    // if the expanded dialog occupies less than 50% of the screen
     int mw = minimumDialogWidth(minimumPageWidth());
-    if (availableSize.width() > 2 * mw) {
-        resize(mw, height());
-    }
+    // expand dialog to minimum size required but do not use more than 80% of screen real estate
+    resize(std::min(int(0.8 * availableSize.width()), mw), height());

```

Comment:
I tend to agree with the linter here, I suggest adding a named constant instead of using a comment, e.g.
```suggestion
    constexpr double maxPercentWidth {0.8}; 
    resize(std::min(int(maxPercentWidth  * availableSize.width()), mw), height());
```

Reply:
Sure! I did introduce it as field of the class + I did the same for the vertical spacing. I also left the comment without referring to  specific value as imho it is nice to state the intent behind this line as it is not the easiest to understand at glance.